ush/chgres_cold2fv3.py

```diff
@@ -1,3 +1,6 @@
+"""
+???
+"""
```

Comment:
@delippi what is this script for?

Reply:
"
The purpose of this code is to convert a cold start file from chgres_cube into a FV3 warm start (restart) file, enabling data blending operators that require a warm start format. The code performs essential transformations, including wind rotation and vertical remapping of atmospheric scalars and winds, ensuring the cold start data aligns with the FV3 model's restart format. Reason: The global 6 h background from chgres_cube is a cold start and must be converted to a warm start file to be able to blend with the 1 h EnKF restarts.
"